scipy/sparse/linalg/dsolve/meson.build

```diff
@@ -0,0 +1,189 @@
+subdir('tests')
+
+if is_windows and meson.get_compiler('cpp', native: true).sizeof('void*') == 4
+  c_args = ['-DNO_TIMER=1']
+else
+  c_args = []
+endif
+c_args += ['-DUSE_VENDOR_BLAS=1']
```

Comment:
Is this correct? Detecting 32 bit windows and assigning C flags accordingly.

Reply:
The void check isn't needed. `win32` means "Windows", not specifically 32-bit. Dumb, I know. See https://stackoverflow.com/questions/2144748/is-it-safe-to-use-sys-platform-win32-check-on-64-bit-python

Reply:
Haha, I took `32` to my heart and added a check for it. I will update this. Though why `32` is suffixed when it doesn't mean anything.

Reply:
> Though why `32` is suffixed when it doesn't mean anything.

Technical debt / software engineers who did not imagine such advanced things as "64-bit computers"